cmd/internal/varinterpolation.go

```diff
@@ -30,6 +30,7 @@ interpolated manifest to STDOUT
 	PreRun: func(cmd *cobra.Command, args []string) {
 		boshManifestFlagViperBind(cmd.Flags())
 		outputFilePathFlagViperBind(cmd.Flags())
+		viper.BindPFlag("variables-dir", cmd.Flags().Lookup("variables-dir"))
```

Comment:
Something like?
```
if err := viper.BindPFlag("variables-dir", cmd.Flags().Lookup("variables-dir")); err != nil {
    log.Fatalf("Unable to add variables-dir flag:%v", err)
}
```

Reply:
I'd ignore that linter warning. If we want to check BindPFlag, we should check all of them, not just one. Sadly we don't currently...